html/storybook-v6/src/stories/button.stories.js

```diff
@@ -0,0 +1,51 @@
+import { withKnobs, select, text, boolean } from '@storybook/addon-knobs';
+import { chevronRightIcon } from '../components/utils/icons';
+import '../components/button/_button.scss';
+import buttonDocs from '../components/button/button.mdx';
+import { withDsm } from '@invisionapp/dsm-storybook';
+
+// This story is written in "Component Story Format" that was introduced in Storybook 5.2
+// https://storybook.js.org/docs/formats/component-story-format/
+export default {
+  title: 'Button',
+  parameters: {
+    decorators: [withKnobs, withDsm],
+    // Module-Level 'in-dsm' configuration (Will apply to all stories inside the module)
+    'in-dsm': { id: '5c86295038392c00aafece83' }
+  }
+};
+
+const buttonIcons = {
+  'chevron-right': chevronRightIcon
+};
+
+const availableIcons = {
+  none: '',
+  'chevron-right': 'chevron-right'
+};
+
+export const simpleButton = () => {
+  const icon = select('icon', availableIcons, availableIcons.none);
+  const disabled = boolean('disabled', false);
+  const children = text('text', 'TEXT');
+
+  return `
+    <div class="dsm-container">
+        <button class="dsm-btn ${icon !== availableIcons.none ? ' dsm-btn--with-icon' : ''}" ${disabled ? 'disabled' : ''}>
+          <div class="dsm-btn__content">${children}</div>
+          ${icon !== availableIcons.none ? buttonIcons[icon] : ''}
+        </button>
+    </div>`;
+};
+
+simpleButton.story = {
+  parameters: {
+    actions: { handles: ['click', '.dsm-btn'] },
```

Comment:
The `storybook/addon-actions` API changed in Storybook 6 so we now pass a "handles" property to the actions parameter.

[Previously we used a `decorated` function and wrapped the HTML in it.](https://github.com/InVisionApp/dsm-storybook-example-library/blob/17cfb0fd17de5a79f4786c12911ad8460cd749c7/html/storybook-v5/src/stories/button.stories.js#L34)

This actions API change is also present in [native-button.stories.js](html/storybook-v6/src/stories/native-button.stories.js).